modules/dnn/perf/perf_einsum.cpp

```diff
@@ -0,0 +1,123 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "perf_precomp.hpp"
+
+namespace opencv_test {
+
+struct EinsumParams {
+    int inputSize;
+    int outputSize;
+    std::string equation;
+    std::vector<MatShape> einsumInpShapes;
+    EinsumParams(std::string equation_, int inputSize_, int outputSize_,  std::vector<MatShape> einsumInpShapes_ = std::vector<MatShape>())
+    {
+        inputSize = inputSize_;
+        outputSize = outputSize_;
+        equation = equation_;
+        einsumInpShapes = einsumInpShapes_;
+    }
+};
+
+static inline void PrintTo(const EinsumParams& params, ::std::ostream* os) {
+     (*os) << "Eqiation=" << params.equation << ", "
+        << "InputSize=" << params.inputSize << ", "
+        << "OutputSize=" << params.outputSize << ", ";
+
+        (*os) << "InputShape={";
+        for(int i = 0; i < params.einsumInpShapes.size(); i++)
+        {
+            (*os) << "{";
+            for(int j = 0; j < params.einsumInpShapes[i].size(); j++)
+            {
+                (*os) << params.einsumInpShapes[i][j] << ((j < params.einsumInpShapes[i].size() - 1) ?  ", " : "");
+            }
+            (*os) << ((i < params.einsumInpShapes.size() - 1) ? "}, " : "}");
+        }
+        (*os) << "}";
+}
+
+// test cases
+static const EinsumParams testEinsumConfigs[] = {
+    // TODO: Add tests with one input after ellips merge
+    {"ij, jk -> ik", 2, 1,  {{2, 3}, {3, 2}}},
```

Comment:
I think that test case is too fast to get reliable perf data

Reply:
When these test were larges @fengyuentau suggested removing large ones. Which one should I follow?

Reply:
It might depends. Please provide a raw output from performance test to see deviation.